src/components/LandingPage/MainComponents/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./styles.css";
+import Button from "../../Common/Button";
+
+function MainComponents() {
+  return (
+    <div className="flex-info">
+      <div className="left-component">
+        <h1 className="track-cypto-heading">Track Crypto</h1>
```

Comment:
Also, fix the font-size here, it is changing with the screen-size and is different in the actual website